### ISSUE TITLE
FAQ page rewrite

### ISSUE THREAD

**Issue by machavity** [SO-Close-Vote-Reviewers/room-meeting-topics#30]
As noted elsewhere, the current FAQ page is somewhat bulky and cumbersome. With nothing but a single list of rules, adding new rules is difficult and has lead to some bulky rules in the middle.

I forked the current site and rewrote the FAQ page to try and address this.

- Uses the same link structure as the existing page so old links won't break
- Has three sections instead of one (Decorum, Moderation and Q&A Interaction)

This proposed page can be viewed here https://github.com/machavity/socvr-website-content/blob/master/pages/faq.md

Thoughts and feedback are welcome

**Comment by Tyler-H** [comment]
I would be in favor of a jQuery UI Tabs layout (horizontal tabs) with different tabs for "expectations for members" "expectations for room owners" "rules on posting *-pls requests", etc.

**Comment by AdrianHHH** [comment]
Having read the proposed new FAQ may I make the following observations:

(1) Generally, it is good.

(2) It would be nice to avoid overlapping numbers for the rules in the different sections. Currently in SOCVR chat people write helpful things like "see FAQ rule 99", which is short and easy to write. The new style FAQ would need something like "see FAQ section Moderation Request Rules, rule 99" to avoid confusion with other sections, such as the 'Q&A Interaction rules' (BTW compare the capitalization of "Rules" in these two).

(3) In Moderation Request Rules, rule 4 I have read the sentence _"So, an average of 10 cv-pls requests per day is just the maximum which is possibly maintainable, if the user is putting the same number of close-votes into cv-pls requests as they are asking from other people by the cv-pls requests that they post."_ several times and find it confusing. I think it is just repeating the information in the previous sentences and so perhaps it could be omitted. Or please consider rewriting it.

(4) The opening sentence in "Why do we need room owners" reads as if the room owners are the cause of the problem. The sentence is _"We really shouldn't need room owners, but because we do [need room owners] we need to establish some guidelines."_ Would it be better if it said something like _"Ideally we shouldn't need room owners, but we sometimes have an overenthusiastic crowd and hence we need to establish some guidelines. The room owners are here to make sure we focus on posts and their quality and not on users."_

(5) The middle paragraph of the "How does the room work" section seems out of place. Perhaps it comes from a time when SOCVR was only about the close queue.

(6) Is the "Why did you invite me to the SOCVR Request Graveyard room" section still appropriate? I do not remember ever being invited there.

(7) Should "Burnination-feed" be added to the "What are all those bots doing there" section?

(8) The last sentence of "Any resources to help" links to the size of the close vote in a neat online graph. The graph appears to be empty.

**Comment by makyen** [comment]
I like having a FAQ that is more organized. My main concern is basically the same as @AdrianHHH's (1): referencing in the room. I think that having multiple things with the same number will result in confusion when people are talking about it in the room. Unless we have a clear way to identify each item, in a way that shows a unique reference to the item when people are viewing the page, I think that we're going to have ongoing communication issues.

**Comment by makyen** [comment]
@AdrianHHH Regarding your (6) and invites to the SOCVR Request Graveyard room: For regular users, that invitation is only issued once, the first time that a request you author is moved to the Graveyard. It's desirable to keep a section mentioning it so new users are not confused and/or we have an explanation to point them to when they are confused. Perhaps we should reword it to make it clear that you only will get that invitation once.

Note: Users that can talk in the room (i.e. ROs and moderators) will get an invitation each time a message they authored is moved there and they are not currently in the room and have been in the room since the last time an invitation was sent.
